catroid/src/main/AndroidManifest.xml

```diff
@@ -243,6 +243,11 @@
         <service android:name=".transfers.MediaDownloadService" />
         <service android:name="com.parrot.freeflight.service.DroneControlService" />
         <service android:name=".utils.StatusBarNotificationManager$NotificationActionService" />
-
-    </application>
+        <service
+            android:name=".ui.cloudmessaging.CustomFirebaseMessagingService">
+            <intent-filter>
+                <action android:name="com.google.firebase.MESSAGING_EVENT"/>
+            </intent-filter>
+        </service>
+        </application>
```

Comment:
:arrow_up: this :wink: not sure why the AS formatter doesn't fix this automatically…
`</application>` should be one tab to the left